ruhimbazabertin/BroadCaster
# #169998939 Fixing api issues

#### What does this PR do?
This PR concerned with correcting Programming logic: 
- Create middlewares to validate flag, 
- Breakdown the testing files into pieces.
-  Correcting errors when a test fails.
 -  Remove dummy data,
 -  Correction of status code.
#### Description of Task to be completed?
 A user should be able to see a good message and status code which is expected after creating a request
#### How should this be manually tested?
- Clone the repository
- Start the server
- Open postman
- Enter the request in postman URL
- User should get a result which concerned with the request sent to the server
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
N/A
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A
